onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -26,16 +26,18 @@ class AttentionCPUBase : public AttentionBase {
                         Tensor* output,            // output tensor
                         int batch_size,            // batch size
                         int sequence_length,       // sequence length
-                        int head_size,             // head size
-                        int hidden_size,           // hidden size
+                        int qk_head_size,          // qk_head_size
+                        int v_head_size,           // head_size
+                        int v_hidden_size,         // hidden_size
+                        const Tensor* extra_add_qk,// extra add in QK. Its size is BxNxSxS
                         OpKernelContext* context) const {
     AllocatorPtr allocator;
     ORT_RETURN_IF_ERROR(context->GetTempSpaceAllocator(&allocator));
 
     auto* tp = context->GetOperatorThreadPool();
 
     int past_sequence_length = 0;
-    Tensor* present = GetPresent(context, past, batch_size, head_size, sequence_length, past_sequence_length);
+    Tensor* present = GetPresent(context, past, batch_size, v_head_size, sequence_length, past_sequence_length);
```

Comment:
@tianleiwu - this is side-effect and v_head_size works. Let me know if any additional test case is needed